pkg/printers/humanprintflags/humanreadable_flags.go

```diff
@@ -14,12 +14,14 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package printers
+package humanprintflags
```

Comment:
That's not acceptable, we can't move just this file. We need to step back and re-think the printers package.

Reply:
> I've talked with David, create a new pancake pkg/kubectl/printers and put there all command facing items there. Currently that'll be all the flags.go files.

Human readable printers need to stay in get.go for now.  It's a snowflake for get.go.  The rest can move.

Reply:
I wouldn't agree, HRP can move the same way as the rest, especially that I'm talking *only* about the flags for printers. HRP will be used in more commands than just get.go